src/Configuration.h

```diff
@@ -0,0 +1,38 @@
+#ifndef __CONFIGURATION_H__
+#define __CONFIGURATION_H__
+
+#include "Rincludes.h"
+#include <string>
+
+class Configuration {
+  public:
+    Configuration(SEXP base_eval, const std::string &analysis_dirpath,
```

Comment:
Shall the `base_eval` be `SEXP const base_eval`?

Reply:
Definitely